src/lib/atoms/KeyValue/KeyValue.tsx

```diff
@@ -1,18 +1,46 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { FC, HTMLAttributes, ReactNode } from 'react';
 import classnames from 'classnames';
 
-// Helpers
+// Configs
+//@ts-ignore
 import { keyValueConfig } from 'configs';
 
 // Components
 import Icon from '../Icon';
 
 // Styles
-import './index.scss';
+import './KeyValue.scss';
+
+export interface IKeyValueProps extends HTMLAttributes<HTMLDivElement> {
+    /**
+     * Label for 'KeyValue'.
+     */
+    label: ReactNode;
+    /**
+     * Value for 'KeyValue'.
+     */
+    value: ReactNode;
```

Comment:
`value` and `label` can be ReactNode | string ,
if ReactNode is covers than please remove string from [here](https://github.com/softconstruct/gene-ui-components/blob/d12740bf910793d29a075b84cce00e75e5aea491/src/lib/atoms/Image/Image.tsx#L44).